content/docs/1.10.1/important-notes/_index.md

```diff
@@ -38,6 +39,76 @@ For the full release note, see [here](https://github.com/longhorn/longhorn/relea
     - [V2 Data Engine Replica Rebuild QoS](#v2-data-engine-replica-rebuild-qos)
     - [V2 Data Engine Volume Expansion](#v2-data-engine-volume-expansion)
 
+## Warning
+
+While the upgrade process typically migrates resources automatically to the new `v1beta2` version, there are scenarios, such as backup system interruptions during the upgrade where some Custom Resource (CR) objects may remain in the deprecated `v1beta1` format.
```

Comment:
`While the upgrade process...`
Need to mention the upgrade is from v1.8.x to v1.9.x